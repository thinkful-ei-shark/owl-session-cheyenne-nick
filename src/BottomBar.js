import React, { Component } from 'react';
import './BottomBar.css';

class BottomBar extends Component {
  render() {
    return (
      <section className="bottom-bar item group-row">
        <div className="actions-header group-row item">
          <button type="button" className="item">{'<'}</button>
          <button type="button" className="item">👍</button>
          <button type="button" className="item">👎</button>
          <button type="button" className="item">✋</button>
          <button type="button" className="item">👏</button>
        </div>

        <div className="actions-header group-row item">
          <button type="button" className="item">📹</button>
          <button type="button" className="item">🔊</button>
          <button type="button" className="item">💻</button>
          <button type="button" className="item">Leave stage</button>
        </div>
      </section>
    );
  }
}

export default BottomBar;