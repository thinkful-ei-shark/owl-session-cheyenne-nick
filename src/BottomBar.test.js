import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import BottomBar from './BottomBar'

describe('<BottomBar />', () => {
    it('renders without crashing', () => {
        // Create a DOM element to render the component into
        const div = document.createElement('div');

        // Render the component
        // If something is wrong it will fail here
        ReactDOM.render(<BottomBar />, div);

        // Clean up
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {

      const bottomBar = renderer
          .create(<BottomBar />)
          .toJSON();
      
      expect(bottomBar).toMatchSnapshot();
    });
});
