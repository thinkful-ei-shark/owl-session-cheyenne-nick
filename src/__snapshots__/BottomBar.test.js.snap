// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BottomBar /> renders the UI as expected 1`] = `
<section
  className="bottom-bar item group-row"
>
  <div
    className="actions-header group-row item"
  >
    <button
      className="item"
      type="button"
    >
      &lt;
    </button>
    <button
      className="item"
      type="button"
    >
      👍
    </button>
    <button
      className="item"
      type="button"
    >
      👎
    </button>
    <button
      className="item"
      type="button"
    >
      ✋
    </button>
    <button
      className="item"
      type="button"
    >
      👏
    </button>
  </div>
  <div
    className="actions-header group-row item"
  >
    <button
      className="item"
      type="button"
    >
      📹
    </button>
    <button
      className="item"
      type="button"
    >
      🔊
    </button>
    <button
      className="item"
      type="button"
    >
      💻
    </button>
    <button
      className="item"
      type="button"
    >
      Leave stage
    </button>
  </div>
</section>
`;
